### DISCUSSION TITLE
Hosted checkout adjustment of PayPal button position

### DISCUSSION THREAD

**Discussion by boostvolt** [swellstores/community#221] [Ideas]
When customers place an order with us, they must already be logged in via our webshop in order to proceed to checkout. As soon as the customer logs in, the cart is notified via the frontend API. However, when the customer opens the hosted checkout, they are taken directly to the "Payment options" section, where only payment methods from e.g. Stripe are displayed, but not the PayPal purchase button. The PayPal button is shown on a previous page "Customer information" which is correctly skipped because the user is already logged in and therefore all necessary information is known. It makes sense that the PayPal button is also placed under the "Payment options" page, otherwise it is not clear to the customer that a PayPal payment option exists. Can this please be adapted accordingly in the hosted checkout or a configuration option be made available via the editor.

Payment options:
<img width="711" alt="Payment options" src="https://user-images.githubusercontent.com/51777660/232566655-afc384cb-fddb-4175-9271-ccdf79dfc7ff.png">

Customer information:
<img width="705" alt="Customer information" src="https://user-images.githubusercontent.com/51777660/232566679-5155ac6f-85a2-409a-a229-5a451b21e7fc.png">

**Comment by ericingram** [comment]
This is a good point, we'll work on adding those methods to the payment page

**Reply by boostvolt** [reply]
Is there an ETA for that? Currently this is a blocking factor for us to use the hosted checkout.

**Reply by logeshswell** [reply]
Currently, we don't have an ETA. We will keep you posted as soon as it's released.
I will move it to Ideas section as it seems to be a good feature request.
